api/src/main/java/com/thesurvey/api/dto/request/UserRegisterRequestDto.java

```diff
@@ -14,24 +15,24 @@
 @AllArgsConstructor
 @NoArgsConstructor
 public class UserRegisterRequestDto {
-
     @NotBlank
     @ApiModelProperty(name = "이름", example = "김진명")
     private String name;
 
-    @Email
-    @NotBlank
+    @Email(message = "유효하지 않은 이메일 형식입니다.")
+    @NotBlank(message = "이메일을 입력해야 합니다.")
     @ApiModelProperty(name = "이메일", example = "test@gmail.com")
     private String email;
 
-    @NotBlank
+    @NotBlank(message = "비밀번호를 확인해주세요.")
+    @Min(value = 8, message = "비밀번호는 최소 8자 이상이어야 합니다.")
```

Comment:
And also please add `@Max` to `25`.

Reply:
Oh.... I apologize for not fixing the issue of using the @Max and @Min annotations on a String type, which only support numeric types....  I will correct it by using the @Size annotation instead.. ;(

Reply:
Very nice! Thank you!